Demo/Application/Features/Ideal/BraintreeDemoIdealViewController.m

```diff
@@ -69,7 +69,7 @@ - (void)startPaymentWithBank {
 
     void (^paymentFlowCompletionBlock)(BTPaymentFlowResult *, NSError *) = ^(BTPaymentFlowResult * _Nullable result, NSError * _Nullable error) {
         if (error) {
-            if (error.code == BTPaymentFlowErrorTypeCanceled) {
+            if (error.code == 5) {
```

Comment:
+1

Reply:
Addressed in the demo!

If we get merchant feedback re: wanting to expose the error codes, we can easily make them public.